testcases/misc_tests/p11sak_test.sh

```diff
@@ -66,6 +69,8 @@ p11sak generate-key rsa 4096 --slot $SLOT --pin $PKCS11_USER_PIN --label p11sak-
 p11sak generate-key ec prime256v1 --slot $SLOT --pin $PKCS11_USER_PIN --label p11sak-ec-prime256v1
 p11sak generate-key ec secp384r1 --slot $SLOT --pin $PKCS11_USER_PIN --label p11sak-ec-secp384r1
 p11sak generate-key ec secp521r1 --slot $SLOT --pin $PKCS11_USER_PIN --label p11sak-ec-secp521r1
+# ibm-dilithium
+p11sak generate-key ibm-dilithium --slot $SLOT --pin $PKCS11_USER_PIN --label p11sak-ibm-dilithium
```

Comment:
How do we know that the slot supports dilithium?  As far as I know, only an EP11 token with only CEX7 or later (and even for CEX7 with a specific FW level) will support dilithium.  So I guess this test will fail quite often in our CI.

Reply:
Yes, this is true. p11sak may check the mechanism list for CKM_IBM_DILITHIUM and issue a msg in case it's not supported. What about to mention the dilithium prereqs in the manpage? Or in the syntax help?

Reply:
I guess what the following commit does should be sufficient:
https://github.com/opencryptoki/opencryptoki/commit/86cda35eec993aa27788d0525d771332775ea16a,


But the call to is_mech_supported() must be added for dilithium keygen similarly to the other places where it is called .